mod/metric_answer_lookup/set/type_plus_right/metric_value/value.rb

```diff
@@ -1,3 +1,5 @@
+card_accessor :unknown
```

Comment:
thanks for that explanation.  let's add `#not stored in database` as a comment on this line.